CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-# dbt_quickbooks version.version
+# dbt_quickbooks v0.17.2
+[PR #155](https://github.com/fivetran/dbt_quickbooks/pull/155) introduces the following updates:
+
+## Bug Fixes
+- Modified `int_quickbooks__bill_payment_double_entry model` to ensuree bill payment accuracy by updating the `converted_amount` logic:
+  - Utilized the line-level `amount` from bill payment lines instead of `total_amount` from bill payments.
+  - Applied `exchange_rate` from the associated bill where available, falling back to the bill payment exchange rate otherwise.
+  - Inverted the sign of the `amount` when associated with a journal entry (to correctly reflect the accounting direction).
+  - Updated joins to link bill payments with related bills and bill payment lines via the `bill_linked_txn` table to support more accurate allocation of payment amounts.
+- These changes improve the integrity and granularity of debit/credit entries for bill payments.
```

Comment:
```suggestion
```
It reads strange that this is it's own bullet. Let's instead add this to the first bullet and let all the sub-bullets specify what the integrity changes are.

Reply:
Moved around.